pattern-library/fonts/edx-icons/edx-icons.css

```diff
@@ -0,0 +1,355 @@
+@font-face {
```

Comment:
This is the CSS generated from Icomoon. This font family is being loaded in `_variables.scss`, but I decided to leave this here so as to require minimal changes to the vendor file. However... we need to modify this file anyway, so I guess we might as well remove this bit.